src/shared/pager.c

```diff
@@ -161,8 +161,8 @@ void pager_close(void) {
                 return;
 
         /* Inform pager that we are done */
-        stdout = safe_fclose(stdout);
-        stderr = safe_fclose(stderr);
+        safe_fclose(stdout);
+        safe_fclose(stderr);
```

Comment:
Yeah, we shouldn't assign `stdout` or `stderr`. In practice it doesn't make much of a difference, but they shouldn't be ever `NULL`.